ios-target/View/Home/TargetFormViewController.swift

```diff
@@ -27,7 +27,12 @@ class TargetFormViewController: ModalViewController {
   }
   
   override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-    if let viewController = segue.destination as? TopicsViewController {
+    if segue.identifier == "matchNotificationSegue",
```

Comment:
how about a switch on `segue.identifier` ?

Reply:
I thought about this too, but I'd do it only if you are planning to add more cases, if not I'd leave it as is.